packages/server-boilerplate/src/orchestrator/api/ApiBuilder.ts

```diff
@@ -47,7 +48,11 @@ export class ApiBuilder {
   // We add handlers at the end so that middlewares and initial routes can be set up first
   private handlers: { add: () => void }[] = [];
 
-  public constructor(transactingConnection: TupaiaDatabase, apiName: string) {
+  public constructor(
+    transactingConnection: TupaiaDatabase,
+    apiName: string,
+    options: { attachModels: boolean } = { attachModels: false },
```

Comment:
Instead of passing in an option, thoughts on adding a function on the builder instead?
```
public attachModels() {
  winston.warn("Best practices say orchestrator servers shouldn't access the db directly, are you sure you need req.models?");
  this.app.use((req: Request, res: Response, next: NextFunction) => {
    req.models = this.models;
  });
  return this;
}
``